src/redux/user/user.reducer.js

```diff
@@ -33,7 +40,37 @@ const userReducer = (state = initialState, action = {}) => {
   case SET_USER_LOADING:
     return {
       ...state,
-      userLoading: true
+      userLoading: action.payload
+    };
+  case STATE_RESET:
+    return {
+      ...state,
+      error: initialState.error,
+      userLoading: initialState.userLoading,
+      userRecovered: initialState.userRecovered,
+      passwordReset: initialState.passwordReset,
+      userRegistered: initialState.userRegistered
+    };
+  case SET_USER_RECOVERED:
+    return {
+      ...state,
+      userRecovered: action.payload,
+      error: false,
+      userLoading: false
+    };
+  case SET_PASSWORD_RESET:
+    return {
+      ...state,
+      passwordReset: action.payload,
+      error: false,
+      userLoading: false
+    };
+  case SET_USER_REGISTERED:
+    return {
+      ...state,
+      userRegistered: action.payload,
+      error: false,
```

Comment:
If `error` contains actual error object, it should have false value of `null`

Reply:
fixed